modules/cf-core/src/methods/state-channel/deploy-state-deposit-holder/controller.ts

```diff
@@ -60,7 +63,7 @@ async function sendMultisigDeployTx(
   signer: Signer,
   stateChannel: StateChannel,
   networkContext: NetworkContext,
-  retryCount: number = 3
+  retryCount: number = 1
```

Comment:
we should have this parameter be configurable by the caller, or retry based on known error messages (cc @rhlsthrm)

also, we need to make sure that in the `deposit` method, if this call fails it will uninstall the `coinBalanceRefund` app within the operation. (may be out of scope of this PR)

Reply:
100% down to retry dependending on error messages.
That said, having the default be anything other than 1 is, in my humble opinion, complete madness.

Reply:
made it an issue -- #599